R/csvTemplate-parser.R

```diff
@@ -277,7 +277,7 @@ templateGen <- function(gh){
     return(this_row)
   
   if(popName == "*" ){
-    if(alias == "*"|| dims == "")
+    if(alias == "*"|| dims %in% c("", NA))
```

Comment:
Kinda surprised that  NA %in% NA returns TRUE, but apparently it does.  R is full of surprises :grin: